src/stories/actionbutton.stories.js

```diff
@@ -0,0 +1,44 @@
+import '../components/atoms/ActionButton/cod-action-button';
+import '../components/atoms/Clickable/cod-clickable';
+import '../components/atoms/Icon/cod-icon';
+import '../components/atoms/CardBody/cod-card-body';
+import '../components/organisms/Card/cod-card';
+import IconStory from './icon.stories';
+
+export default {
+  component: 'cod-action-button',
+  title: 'Components/Atoms/ActionButton',
+  // 👇 Creates specific argTypes
+  argTypes: {
+    buttonColor: {
+      options: ['btn-primary', 'btn-secondary', 'btn-success'],
+      control: 'select',
+    },
+    icon: IconStory.argTypes.icon,
+    title: {
+      control: 'text',
+    },
+    body: {
+      control: 'text',
+    },
+  },
+  args: {
+    buttonColor: 'btn-primary',
+    title: 'Do Something',
+    body: 'Like click on this card',
+    icon: 'house-fill',
+  },
+};
+
+// Template
+const Template = (args) => {
+  const aButton = document.createElement('cod-action-button');
+  aButton.setAttribute('btn-color', args.buttonColor);
+  aButton.setAttribute('icon', args.icon);
+  aButton.setAttribute('title', args.title);
+  aButton.setAttribute('body', args.body);
```

Comment:
It might be better to setup the component using `slots` rather than attributes. It will probably make the implementation easier. The `body` field on drupal will export mark up.

Reply:
@jedgar1mx , I intentionally used an attribute here because that will force users of the button to not try and shove too much inside the button. These buttons aren't really designed to have rich, formatted content inside. The 'body' is meant to be a simple sentence or few words.

With that in mind, let me know if you still think slots is a better approach here.

Reply:
Using slot for the body field is probably a better approach to accommodate the old content.